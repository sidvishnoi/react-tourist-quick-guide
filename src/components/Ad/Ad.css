.Ad {
  background: #eee;
  margin: 0.5em auto;
  padding: 1em;
  text-align: center;
}
