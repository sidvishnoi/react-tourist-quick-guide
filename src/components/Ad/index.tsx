import * as React from 'react';
import './Ad.css';

export default function Ad() {
  return <div className="Ad">Advertisement Space</div>;
}
