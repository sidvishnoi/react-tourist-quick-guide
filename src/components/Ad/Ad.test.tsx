import React = require('react');
import * as renderer from 'react-test-renderer';

import Ad from '.';

describe('Component - Ad', () => {
  it('matches snapshot', () => {
    const tree = renderer.create(<Ad />);
    expect(tree).toMatchSnapshot();
  });
});
