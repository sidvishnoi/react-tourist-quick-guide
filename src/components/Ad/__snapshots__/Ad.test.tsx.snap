// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - Ad matches snapshot 1`] = `
<div
  className="sc-bdVaJa hDTFWN"
>
  Advertisement Space
</div>
`;
